proposals/nnnn-access-level-on-imports.md

```diff
@@ -0,0 +1,321 @@
+# Access-level on import statements
+
+* Proposal: [SE-NNNN](NNNN-access-level-on-imports.md)
+* Author: [Alexis Laferrière](https://github.com/xymus)
+* Review Manager: TBD
+* Status: **Awaiting review**
+* Implementation:
```

Comment:
As discussed on the forums, let's replace these links with just a reference to the necessary experimental feature flag.